package.json

```diff
@@ -136,9 +136,7 @@
     "package": {
       "command": "cd editor/vscode && vsce package",
       "dependencies": [
-        "style-dictionary",
-        "types",
```

Comment:
Do I leave these as dependencies even if they're run beforehand?

Reply:
if package is used elsewhere, then yes. if it's only used in the CI script, we can leave it as is. approving now, merge after you check. THANKS!